### PR TITLE
fix #2 Add more docs

### LINKED ISSUE #2: Add more docs to the readme
Currently the Readme has no instructions at all, I will add some more useful instructions to make things a little easier for the new comers.

### PR DESCRIPTION
**Pull request by Ahmed-Ayman**

### CODE REVIEW

File: VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ bootstrap_barrio_button: 1
 bootstrap_barrio_button_size: ''
 bootstrap_barrio_button_outline: 0
 bootstrap_barrio_navbar_toggle: navbar-toggleable-md
-bootstrap_barrio_navbar_container: 1
+bootstrap_barrio_ngit@github.com:Ahmed-Ayman/vartheme_bs4.gitavbar_container: 1
```
Comment (Natshah) [inline VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml]: Strange to add Ahmed-Ayman/vartheme_bs4 in the setting file

File: VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ bootstrap_barrio_button: 1
 bootstrap_barrio_button_size: ''
 bootstrap_barrio_button_outline: 0
 bootstrap_barrio_navbar_toggle: navbar-toggleable-md
-bootstrap_barrio_ngit@github.com:Ahmed-Ayman/vartheme_bs4.gitavbar_container: 1
+bootstrap_barrio_navbar_container: 1
```
Comment (Natshah) [inline VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml]: Ok removed at this point

File: README.md [hunk 3]
```diff
@@ -14,5 +14,12 @@ A SASS base theme for [Varbase](https://www.drupal.org/project/varbase) standard
 
 To Create a subtheme from Vartheme BS4, you need to copy the VARTHEME_BS4_SUBTHEME folder to `web/themes/custom/` then rename the directory to your theme name, along with all the strings underneath it. or simply if you have bash on your system you can use our automated scrip that will do all of this for you.
 
-[Here's the script link](https://github.com/Vardot/vartheme_bs4/tree/8.x-6.x/scripts).
+[Here's the script link](https://github.com/Vardot/vartheme_bs4/tree/8.x-6.x/scripts). it's also doing a bit of a magic for you like downloading the necessary npm modules. after finishing it you should already have gulp watch up and running.
```
Comment (Natshah) [inline README.md:17]: Agree on this

File: VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml (hunk 1 above)
Comment (Ahmed-Ayman) [reply VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml]: Ops, sounds like a typo. I'll fix it.

File: VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml (hunk 2 above)
Comment (Ahmed-Ayman) [reply VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml]: It was a typo yes

File: VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml (hunk 1 above)
Comment (Ahmed-Ayman) [reply VARTHEME_BS4_SUBTHEME/config/install/VARTHEME_BS4_SUBTHEME.settings.yml]: This was a typo that I've fixed later in the fix typo commit.

File: README.md (hunk 3 above)
Comment (Ahmed-Ayman) [reply README.md:17]: Awesome.

**Pull request closed without merging**